desktop.css

```diff
@@ -0,0 +1,330 @@
+@media all and (min-width: 501px) {
+  body {
+    align-items: center;
+  }
+
+  .menu-items {
+    display: flex;
+    justify-content: space-between;
+    width: 250px;
+    height: 40px;
+    margin-top: 2rem;
+    align-self: flex-end;
+  }
+
+  .menu-items a {
+    color: #344563;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    text-decoration: none;
+  }
+
+  .nav-container {
+    display: flex;
+    justify-content: space-around;
+    background-color: white;
+    flex-direction: row;
+  }
+
+  .logoholder {
+    order: 0;
+    margin-left: 12%;
+  }
+
+  .mylogo {
+    position: static;
+    width: 76px;
+    height: 20px;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 18px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #6070ff;
+    flex: none;
+    flex-grow: 0;
+    margin-top: 2rem;
+    text-decoration: none;
+  }
+
+  .mobile-only {
+    display: none;
+  }
+
+  .header-container {
+    background-image: url("./src/Header\ bg.svg");
+    display: flex;
+    flex-direction: column;
+    background-repeat: no-repeat;
+    height: 846px;
+    width: 100%;
+    background-color: white;
+    border-radius: 0 0 0 100px;
+    background-size: 97% 85%;
+    background-position: center;
+  }
+
+  .headerholder {
+    width: auto;
+    margin-top: 18rem;
+    margin-left: 18.05%;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .head {
+    margin-top: 1rem;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 48px;
+    line-height: 60px;
+    width: 30%;
+    margin-bottom: 0;
+  }
+
+  .headtext {
+    width: 63.9%;
+    margin-left: 18.05%;
+    margin-right: 18.05%;
+    margin-top: 1rem;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    flex: none;
+    flex-grow: 0;
+    margin-bottom: 0;
+  }
+
+  .supporttext {
+    margin-left: 18.05%;
+    margin-top: 1.4%;
+  }
+
+  .suptext {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  .social {
+    display: flex;
+    order: 3;
+    justify-content: space-between;
+    width: 170.17px;
+    margin-top: 10px;
+  }
+
+  .portfolio-card {
+    display: flex;
+    border: 1px solid #dfe1e6;
+    border-radius: 24px;
+    align-items: center;
+    padding: 0.2rem 1rem;
+    justify-content: space-between;
+    margin-top: 10%;
+    width: 65%;
+    gap: 4%;
+  }
```

Comment:
Your project cards don't quite look like those from the Figma design. Take a closer look at the template design and fix the margins/paddings.

<img width="1047" alt="Screen Shot 2022-07-07 at 15 32 45" src="https://user-images.githubusercontent.com/82426682/177866314-d876e364-aba9-4535-8d15-ad3c7dc759dd.png">